tests/js/spec/views/__snapshots__/ownershipInput.spec.jsx.snap

```diff
@@ -1222,12 +1221,10 @@ exports[`Project Ownership Input renders 1`] = `
 path:src/example/pipeline/* person@sentry.io #infra
 url:http://example.com/settings/* #product
 tags.sku_class:enterprise #enterprise"
-      rows={1}
       spellCheck="false"
       value="new"
     >
-      <TextareaAutosize
-        async={false}
+      <ForwardRef
```

Comment:
maybe you could name the anonymous function  "TextareaAutosize" and/or give it a display name 😉 because it makes debugging easier

Reply:
Nice! It was tough to think of a name for it

Reply:
this for some reason didn't pick up the new name and the other did?

Reply:
I don't understand what you mean

Reply:
the forwardRef returns an anonymous func (without name). That is the reason. you can give the same name of the component 😉

Reply:
figured it out 😄